src/renderer/components/test-utils/get-application-builder.tsx

```diff
@@ -120,6 +121,21 @@ export const getApplicationBuilder = () => {
   rendererDi.override(storesAndApisCanBeCreatedInjectable, () => true);
   mainDi.override(clusterStoreInjectable, () => clusterStoreStub);
 
+  mainDi.override(randomBytesInjectable, () => {
```

Comment:
Override has side-effect. Could this override return a static value instead? The answer could be no if some code that is uninteresting in the general use-case requires differing return values in order to not blow up.

Reply:
Will try.

Reply:
I think this can probably be static (for the time being at least) since the only use case is authentication.